src/sources/postgresql_metrics.rs

```diff
@@ -258,20 +284,26 @@ impl DatnameFilter {
         let mut conditions = "SELECT * FROM pg_stat_database_conflicts".to_owned();
         match self {
             Self::Include { databases, .. } => {
```

Comment:
I missed this before, but is it possible to combine both `include` and `exclude`?

For example, could I write:

```
include = ["foo.*"]
exclude = ["foobar"]
```

To fetch all databases starting with `foo` except `foobar`?

Reply:
We do not combine both because in the previous case of `eq` operation (`=`) is did not make sense but with changing to regexp I think we should =\